src/EventHandlers.ts

```diff
@@ -24,98 +23,92 @@ CDAIContract_LiquidateBorrow_loader(({ event, context }) => {
   let liquidatorAddress: string = event.params.liquidator.toString();
   let liquidatedAddress: string = event.params.borrower.toString();
   let ctokenAddress: string = event.params.cTokenCollateral.toString();
-  let tokenWonID: string = liquidatorAddress.concat(ctokenAddress);
-  let tokenLostID: string = liquidatedAddress.concat(ctokenAddress);
-
-  context.liquidatorAccount.load(liquidatorAddress);
-  context.liquidatedAccount.load(liquidatedAddress);
-  context.tokenWon.load(tokenWonID, { loaders: { loadLiquidatorAccount: true } });
-  context.tokenLost.load(tokenLostID, { loaders: { loadLiquidatedAccount: true } });
+  let tokenwonID: string = liquidatorAddress.concat(ctokenAddress);
+  let tokenlostID: string = liquidatedAddress.concat(ctokenAddress);
+  context.liquidatoraccount.load(liquidatorAddress);
+  context.liquidatedaccount.load(liquidatedAddress);
+  context.tokenwon.load(tokenwonID, {
+    loaders: { loadLiquidatorAccountX: true },
+  });
+  context.tokenlost.load(tokenlostID, {
+    loaders: { loadLiquidatedAccountX: true },
+  });
 });
 
 CDAIContract_LiquidateBorrow_handler(({ event, context }) => {
```

Comment:
Sorry to chop all your logic here into a comment but I havent gone through the actual indexer logic and didnt go through and change all the entity name capitalization changes

Reply:
No worries at all, it's my job to de-tangle all that.  I just appreciate you getting it running again!